lib/src/meta_extensions.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:turf/src/meta.dart' as meta;
+
+import '../helpers.dart';
+
+extension GeomEachGeoJSONObject on GeoJSONObject {
+  void geomEach(meta.GeomEachCallback callback) {
+    meta.geomEach(this, callback);
+  }
+}
+
+extension GeomEachGeometryObject on GeometryObject {
+  void geomEach(
+    meta.GeomEachCallback callback, {
+    Map<String, dynamic>? featureProperties,
+    BBox? featureBBox,
+    dynamic featureId,
+    int? featureIndex,
+  }) {
+    try {
+      meta.forEachGeomInGeometryObject(
+        this,
+        callback,
+        featureProperties,
+        featureBBox,
+        featureId,
+        featureIndex ?? 0,
+      );
+    } on meta.ShortCircuit {
+      return;
+    }
+  }
+}
+
+extension GeomEachFeature on Feature {
+  void geomEach(meta.GeomEachCallback callback, [int? featureIndex]) {
+    try {
+      meta.forEachGeomInFeature(
+        this,
+        callback,
+        featureIndex ?? 0,
+      );
+    } on meta.ShortCircuit {
+      return;
+    }
+  }
+}
+
+extension GeomEachFeatureCollection on FeatureCollection {
```

Comment:
I'm still not sure I understand why the specific implementations for features and geometries are necessary... Doesn't the call at the abstract GeoJSONObject cover all these cases within the implementation of geomEach?

Reply:
I thought a little bit about it. I did it because of additional optional parameters. But I don't think tey are needed really